stories/Components/UIcomponents/Hero/PageHero/PageHero.jsx

```diff
@@ -5,35 +5,45 @@ import BackgroundVideo from '../../../../assets/video/video_sample.mp4';
 import { Video } from '../../../../Atom/Video/Video';
 import { Heading } from '../../../../Atom/Typography/Heading/Heading';
 import { Breadcrumbcomponent } from '../../../Navigationcomponents/Breadcrumbs/Breadcrumbs';
+import { CtaButton } from '../../Buttons/CtaButton/CtaButton';
 
-export const PageHero = ({
-  data, title, content, variant, imgsrc, imgalt,
-}) => {
+export function PageHero({
+  data,
+  title,
+  subtitle,
+  content,
+  variant,
+  imgsrc,
+  imgalt,
+  cta,
+  ...args
+}) {
   useEffect(() => {
     expandToSize('.pagehero-full');
   }, []);
+  const mainClasses = variant === 'no_background' ? 'pagehero-full no-background-media' : 'pagehero-full';
+  const textColor = variant === 'no_background' ? 'Black' : 'White';
+  const textColorClass = variant === 'no_background' ? 'color-black' : 'color-white';
+
   return (
-    <div className="pagehero-full">
-      {variant === 'video'
-        ? (
-          <Video src={BackgroundVideo} width="100%" height="100%" />
-        )
-        : (
-          <div className="overlay-grey">
-            <img src={imgsrc} alt={imgalt} />
-          </div>
-        )}
-      <Breadcrumbcomponent data={data} Color="White"/>
+    <div className={mainClasses}>
+      {variant === 'video' && (
+        <Video src={BackgroundVideo} width="100%" height="100%" />
+      )}
+      {variant === 'image' && (
+        <div className="overlay-grey">
+          <img src={imgsrc} alt={imgalt} />
+        </div>
+      )}
+      <Breadcrumbcomponent data={data} Color={textColor} />
       <div className="pagehero-content  medium-offset-1">
         <div className="content-box">
-          {content && <Heading type="4" className="color-white" label={content} dataViewport="true"/>}
-          {title && <Heading type="2" className="color-white" label={title} dataViewport="true"/>}
+          {args.outline_enabled && content && <Heading type="4" className={textColorClass} label={content} dataViewport="true" />}
```

Comment:
please avoid using "color-black", "color-white" classes on individual elements inside Hero. That styling should be defined at the container level, something like .pagehero-content or its .content-box inside.

Reply:
This is cleaned up. Text color related classes are moved to the "content-box" parent, to catch all content.